contracts/liquidator/Liquidator.sol

```diff
@@ -186,19 +233,19 @@ contract Liquidator is IUniswapV3FlashCallback, PeripheryImmutableState, Periphe
             swapToken = weth;
             poolFee = getPoolConfigForAsset(weth).fee;
 
-            TransferHelper.safeApprove(weth, address(swapRouter), swapAmount);
+            TransferHelper.safeApprove(weth, address(uniswapRouter), swapAmount);
         }
 
         // Swap asset or received ETH to base asset
-        uint256 amountOut = swapRouter.exactInputSingle(
+        uint256 amountOut = uniswapRouter.exactInputSingle(
             ISwapRouter.ExactInputSingleParams({
                 tokenIn: swapToken,
                 tokenOut: baseToken,
                 fee: poolFee,
                 recipient: address(this),
                 deadline: block.timestamp,
                 amountIn: swapAmount,
-                amountOutMinimum: 0,
+                amountOutMinimum: amountOutMin,
```

Comment:
specifying an `amountOutMinimum` helps throw an error at the preferable place in the code, since we'll get get errors specifying that we didn't receive enough in return

Reply:
cool